e2e/test/specs/travelsearch.e2e.ts

```diff
@@ -81,9 +83,27 @@ describe('Travel search', () => {
       // Check travel time
       const travelTimeInDep =
         await TravelsearchDetailsPage.travelTime.getText();
-      expect(travelTimeInDep).toContain(travelTime.toString());
+      expect(timeIsEqual(travelTimeInDep, travelTime)).toEqual(true);
 
       await NavigationHelper.back();
+
+      /**
+       * Check if two times are equal
+       * Note! Simple check for this search wher travel time is in minutes
+       * @param time1 String that contains some digits to compare
+       * @param time2 String that contains some digits to compare
+       * @param allowedDiff Allow a time difference
+       */
+      function timeIsEqual(
```

Comment:
Could consider renaming to e.g. `timeIsCorrect` or `timeIsMatching` since it is not a strict equals check. In theory it could be a mistake that matches..

Reply:
I agree! Fixed.